histomicstk/utils/del2.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 
 
-def Del2(X):
+def del2(X):
```

Comment:
Can we use [scipy.ndimage.filters.laplace](https://docs.scipy.org/doc/scipy-0.16.0/reference/generated/scipy.ndimage.filters.laplace.html) for this?